bubblewrap.c

```diff
@@ -149,6 +151,8 @@ usage (int ecode)
            "	--bind SRC DEST		     Bind mount the host path SRC on DEST\n"
            "	--dev-bind SRC DEST	     Bind mount the host path SRC on DEST, allowing device access\n"
            "	--ro-bind SRC DEST	     Bind mount the host path SRC readonly on DEST\n"
+           "	--exec-label LABEL           Execq Label to run the sandbox with\n"
```

Comment:
We are referring to this as a sandbox. So I am changing to 

"Exec label for the sandbox\n"